src/makielayout_integration.jl

```diff
@@ -92,6 +105,72 @@ function layoutplot!(scene, layout, ts::Algebraic)
             @warn "Automated legend was not possible due to $e"
         end
     end
+    
+    ax1 = axs[end,1]
```

Comment:
Due to my use of fake data, I didn't realize that I got the spanned x and y labels wrong. This is now fixed.

Now I take them from one `ax1`.

It would probably be preferable to take them directly from `split(trace.value)`?

Reply:
Did you experiment with the scenario where not all pairwise group combinations exist? That is, it's not a perfect grid but some plots are missing. If we want to support that, we'd need to make sure that `axs[end, 1]` does the right thing there.